lxd/instance/drivers/driver_qemu.go

```diff
@@ -8895,15 +8898,16 @@ func (d *qemu) deviceDetachUSB(usbDev deviceConfig.USBDeviceItem) error {
 
 // Block node names and device tags may only be up to 31 characters long, so use a hash if longer.
 func (d *qemu) generateQemuDeviceName(name string) string {
-	if len(name) > 27 {
+	maxNameLenght := qemuDeviceNameMaxLenght - len(qemuDeviceNamePrefix)
```

Comment:
Why is the prefix not included in the max name length comparison?

Reply:
So you're just deriving the 27 char from the constant of 31?

Reply:
yes, I could just hardcode the 27 in but I thought it looked better deriving the limit from the defined constants.

Reply:
Thanks makes sense, please include this sort of thing in commit messages as it helps the reviewer understand the reason behind the changes. Ta